src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayList.js

```diff
@@ -108,9 +114,12 @@ export default class OverlayList extends React.PureComponent<Props> {
     handleBackropClick = this.requestClose;
 
     render() {
-        const dimensions = this.dimensions;
-        const style = dimensions ? OverlayListPositioner.dimensionsToStyle(dimensions) : {opacity: '0'};
-        this.scrollTop = dimensions ? dimensions.scrollTop : 0;
+        let style = {opacity: '0'};
```

Comment:
Is that a good idea? This way it might still block clicks that might be behind that that element, doesn't it?